### PR TITLE
[Math] Add degrees to radians method to MDCMath

### PR DESCRIPTION
**Pull request by jgunaratne**
Add degrees to radians method to MDCMath

### CODE REVIEW

File: components/private/Math/src/MDCMath.h [hunk 1]
```diff
@@ -30,6 +30,10 @@ static inline CGFloat MDCFabs(CGFloat value) {
 #endif
 }
 
+static inline CGFloat MDCDegreesToRadians(CGFloat degrees) {
+  return degrees * M_PI / 180.f;
```
Comment (willlarche) [inline components/private/Math/src/MDCMath.h]: Could you make it double if cgfloat is double and float if not?

**Review by willlarche — DISMISSED** [review]

File: components/private/Math/src/MDCMath.h (hunk 1 above)
Comment (ianegordon) [reply components/private/Math/src/MDCMath.h]: Same comment from #2016 
With the explicit 180.f there is a cast if CGFLOAT_IS_DOUBLE.

#if CGFLOAT_IS_DOUBLE
return degrees * M_PI / 180.0;
#else
return degrees * M_PI / 180.f;
#endif

**Review by ianegordon — DISMISSED** [review]
Comment Inline.

**Review by ianegordon — APPROVED** [review]
LGTM.
Please merge once Travis is happy.

**Review by randallli — APPROVED** [review]

**Review by willlarche — APPROVED** [review]

**Pull request merged**